web/src/components/profile/ProfileNoteForBusinessesFormedOutsideNavigator.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { isFormedOutsideNavigator } from "@/lib/utils/helpers";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement } from "react";
+
+interface Props {
+  business: Business | undefined;
+}
+
+export const ProfileNoteForBusinessFormedOutsideNavigator = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  return isFormedOutsideNavigator(props.business) ? (
+    <Alert variant="note" borderSmall borderRight>
+      {Config.profileDefaults.default.noteForBusinessesFormedOutsideNavigator}
+    </Alert>
+  ) : (
+    <></>
```

Comment:
[pebble] We try to avoid ternaries in JSX, the nesting makes it hard to read. I think this would do a guard clause here.